templates/agent-conf.d/http_check.yaml.erb

```diff
@@ -32,6 +32,9 @@ instances:
 <% if instance['window'] -%>
     window: <%= instance['window'] %>
 <% end -%>
+<% if instance['reverse_content_match'] -%>
+    reverse_content_match: '<%= instance["reverse_content_match"] %>'
```

Comment:
could you please remove the single quotes here around: `<%= instance["reverse_content_match"] %>`, we don't need those :)

Reply:
Good catch! Fixed!